lib/DDG/Spice/Quandl/HomeValues.pm

```diff
@@ -0,0 +1,90 @@
+package DDG::Spice::Quandl::HomeValues;
+
+use DDG::Spice;
+use Text::Trim;
+
+# meta data
+# Initially this is will work with zip codes, but will expand
+# to other region identifiers
+# 
+primary_example_queries "27514 home values";
+secondary_example_queries "one bedroom houses 27514";
+description "Home values for a given region";
+name "Home Values";
+code_url "https://github.com/brianrisk/zeroclickinfo-spice";
+icon_url "https://www.quandl.com/favicon.ico";
+topics "economy_and_finance";
+category "finance";
+attribution web => ["https://www.quandl.com", "Quandl"],
+            twitter => "quandl";
+            
+# load our trigger phrases
+my @trigger_lines = share('home_values_triggers.txt')->slurp;
+# hash to associate phrases with URL codes
+my %trigger_phrases = ();
+# array to preserve the order of the trigger phrases from the text file
+my @trigger_keys;
+foreach my $line (@trigger_lines) {
```

Comment:
We can simply almost all of this down to a single line if we convert `triggers.txt` to a YAML file. E.g:

```yaml
total assets: ASSETS
current assets: ASSETSC
...
```

See: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/SalesTax.pm#L30, https://github.com/duckduckgo/zeroclickinfo-spice/blob/9d370de370b4094caa8802dac896b6c337ff619e/share/spice/sales_tax/zipcodes.yml